src/cpp/DataBrokerConfiguration.cpp

```diff
@@ -143,18 +161,50 @@ bool DataBrokerConfiguration::load_configuration_file(
         // UDP
         if (config_node["udp"])
         {
-            configuration.udp = config_node["udp"].as<bool>();
-            if (configuration.ros)
+            configuration.wan_configuration.udp = config_node["udp"].as<bool>();
+            if (configuration.local_configuration.ros)
             {
                 logInfo(DATABROKER_CONFIGURATION, "Using UDP transport");
             }
         }
 
+        // TLS
+        if (config_node["tls"])
+        {
+
+            if (config_node["tls"]["private_key"] &&
+                    config_node["tls"]["password"] &&
+                    config_node["tls"]["dh"] &&
+                    config_node["tls"]["ca"] &&
+                    config_node["tls"]["verify_ca"])
+            {
+                configuration.wan_configuration.tls_private_key = config_node["tls"]["private_key"].as<std::string>();
+                configuration.wan_configuration.tls_password = config_node["tls"]["password"].as<std::string>();
+                configuration.wan_configuration.tls_dh_params = config_node["tls"]["dh"].as<std::string>();
+                configuration.wan_configuration.tls_ca_cert = config_node["tls"]["ca"].as<std::string>();
+                configuration.wan_configuration.tls_cert = config_node["tls"]["verify_ca"].as<std::string>();
+                configuration.wan_configuration.tls = true;
+            }
+            else
+            {
+                if (verbose)
```

Comment:
The verbose option was removed from the databroker options so remove the `ìf` clause here and below.

Reply:
This verbose is used to avoid the warning in case the default file does not exist
I'll remove it from TLS